src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -151,6 +153,13 @@
       (cond
         (:show-import-mc-modal? @state) [ConfigurationImportForm {:selected-method-config (:selected-method-config @state)
                                                                   :workspace-id (:workspace-id props)
+                                                                  :on-back
+                                                                  (fn []
+                                                                    (swap! state dissoc
+                                                                      :show-import-mc-modal?
+                                                                      :selected-method-config)
+                                                                    (swap! state assoc
+                                                                      :loaded-import-confs? true))
```

Comment:
We don't need to much with `:loaded-import-confs?` in order to do this window switch.  If you get rid of this swap and also the `:loaded-import-confs? false` a few lines below, this still works.  Really, what we're switching on is `:show-import-mc-modal?` (which is poorly named now); whether or not the import confs have been loaded doesn't change.